src/style.css

```
@@ -1 +1,105 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: normal;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Heavy.ttf') format('truetype');
+  font-weight: bold;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir.ttc') format('truetype');
+  font-weight: 400;
+  font-style: normal;
+}
+
+body {
+  font-family: Avenir, Arial, sans-serif;
+}
+
+.form {
+  font-family: Avenir, sans-serif;
+  display: block;
+  margin-top: 20px;
+  outline: none;
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+  width: 100%; /* Розтягує форму на всю ширину контейнера */
```

Remove all comments, it's redundant here
```suggestion
  width: 100%; 
```